lua/ulx/modules/sh/cfc_warn_after.lua

```diff
@@ -1,23 +1,26 @@
+CFCUlxCommands.warnAfter = CFCUlxCommands.warnAfter or {}
+local cmd = CFCUlxCommands.warnAfter
+
 local CATEGORY_NAME = "Utility"
 
-local function warnban( calling_ply, target_ply, minutes, reason )
+function cmd.warnban( calling_ply, target_ply, minutes, reason )
```

Comment:
```suggestion
function cmd.warnBan( calling_ply, target_ply, minutes, reason )
```
Would also need to change where this is used